entry/build-profile.json5

```diff
@@ -0,0 +1,13 @@
+{
+  "apiType": 'faMode',
+  "buildOption": {
+  },
+  "targets": [
+    {
+      "name": "default",
+    },
+    {
+      "name": "ohosTest",
+    }
+  ]
+}
```

Comment:
delete package-lock.json file

Reply:
Do you need to delete the file completely?